src/query/execute_schema.c

```diff
@@ -8854,6 +8862,36 @@ execute_create_select_query (PARSER_CONTEXT * parser, const char *const class_na
   return error;
 }
 
+/* TODO: When creating a table, replication information can be obtained from the parse tree (PT), 
+but when adding a constraint to a column with ALTER TABLE, it has to be retrieved from the schema manager (SM). 
+If the time required to patch the SM is small, the same patch-based approach can be applied consistently. 
+*/
+bool
+ha_fk_replication_violation (DB_OBJECT * class_obj, bool repl)
```

Comment:
nit: 함수 이름에서 복제 옵션에 대한 확인을 한다는 맥락이 있으면 좋을 것 같습니다.

```suggestion
ha_check_fk_replication_violation (DB_OBJECT * class_obj, bool repl)
```

Reply:
@hgryoo  반영했습니다.